microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-106.md

```diff
@@ -0,0 +1,179 @@
+---
+title: What's new in DevTools (Microsoft Edge 106)
+description: "Introducing the Command Palette. Filter and automatically fix code issues in DevTools for Visual Studio Code. DevTools now uses less disk space when caching sourcemaps. Chromium color themes have been deprecated. Accessibility improvements in Focus Mode and high contrast mode. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 09/29/2022
+---
+# What's New in DevTools (Microsoft Edge 106)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Introducing the Command Palette
+
+<!-- Subtitle: Your favorite commands in one place. -->
+<!--           Your favorite browser and DevTools commands in one place. -->
+
+Command Palette makes it possible to access browser management and developer tools with just one keyboard shortcut: `Ctrl`+`Q`.  Let us know what commands we should add next!
+
+To access the available browser management commands and DevTools commands, press `Ctrl`+`Q`.  To filter the commands, start typing:
+
+![Microsoft Edge Command Palette](./devtools-106-images/command-palette.png)
+
+To list or filter the available DevTools commands, press `Ctrl`+`Q`, and then enter the greater-than (**>**) character:
+
+![Microsoft Edge Command Palette entering `>` to select DevTools commands](./devtools-106-images/command-palette-devtools.png)
+
+*  In Microsoft Edge 108 and later, the shortcut is `Ctrl`+`Q`.
```

Comment:
What if we cut this whole section and just did an update when Edge 108 is promoted to Stable with the new keyboard shortcut? This is What's New 106 so I think we should be focusing on the state of the product in 106 itself. @erdraud, please consider.

Reply:
The shortcut is shown in the flags page under Command Palette.  
* For earlier minor releases of 107, the flags page shows Ctrl+Shift+Spacebar.  
* For later minor releases of 107, the flags page shows Ctrl+Q.  
It's somewhat possible to reword this by leveraging that version-specific flags-page wording.

Reply:
copy/paste of the above thread's comment:
done. In What's New 106, limited this section to 106 vs. previous versions, not later versions. See also the linked regular article in this PR, which is broader. Added important point & screenshot here, about where to view what the shortcut (in a given version) is. @zoherghadyali @erdraud